addon/ng2/tasks/format.js

```diff
@@ -0,0 +1,23 @@
+/* jshint node: true */
+'use strict';
+
+var Promise = require('ember-cli/lib/ext/promise');
+var Task = require('ember-cli/lib/models/task');
+var exec = Promise.denodeify(require('child_process').exec);
```

Comment:
Yeah, it was more of a theoretical question, because they are surely already imported elsewhere.
Thx for the response.

--
> I'm also not a big fan of conditional imports myself

Interesting (considering how `chalk` is imported below)... :stuck_out_tongue: